packages/cli/src/util/parse-target.ts

```diff
@@ -23,11 +23,9 @@ export default function parseTarget<FlagName extends string>({
   }
 
   if (typeof targetFlagValue === 'string') {
-    const lowerCaseTarget = targetFlagValue.toLowerCase();
-    output.debug(`Setting target to ${lowerCaseTarget}`);
-    return lowerCaseTarget;
+    return targetFlagValue
```

Comment:
Needs a changeset. Also, I'm curious why the pre-commit formatter didn't run? (missing semicolon)

Reply:
yeah I just found that too 🤔 

I'll get the changeset made in the next commit because I forgot about that 🙃